tests/integ/test_column_names.py

```diff
@@ -417,18 +436,51 @@ def add_one(x: int) -> int:
     perm_func_name = (
         f'"{special_chars}{Utils.random_name_for_temp_object(TempObjectType.FUNCTION)}"'
     )
+    database_name = session.get_current_database()
+    schema_name = session.get_current_schema()
+    full_temp_func_name = f"{database_name}.{schema_name}.{temp_func_name}"
+
     stage_name = Utils.random_stage_name()
-    try:
+    if not local_testing_mode:
```

Comment:
I believe with Adams change to Utils this check is no longer needed.

Reply:
minor: I merged my PR, you can remove this if check now